packages/ckeditor5-highlight/tests/highlightui.js

```diff
@@ -78,11 +78,19 @@ describe( 'HighlightUI', () => {
 		it( 'button has the base properties', () => {
 			const button = dropdown.buttonView;
 
-			expect( button ).to.have.property( 'tooltip', 'Highlight' );
+			expect( button ).to.have.property( 'label', 'Highlight' );
+			expect( button ).to.have.property( 'tooltip', true );
 			expect( button ).to.have.property( 'icon', markerIcon );
 			expect( button ).to.have.property( 'isToggleable', true );
 		} );
 
+		it( 'button arrow has the base properties', () => {
+			const button = dropdown.buttonView.arrowView;
+
+			expect( button ).to.have.property( 'label', 'Highlight' );
+			expect( button ).to.have.property( 'tooltip', true );
+		} );
+
```

Comment:
Obsolete per my comment in [codeblockui.js](https://github.com/ckeditor/ckeditor5/pull/11859/files?show-viewed-files=true&file-filters%5B%5D=#diff-a75a67319b38c5dc7c27b30eeaff970892cdd73e01b28920a569d161458ea8c7).